package.json

```diff
@@ -37,7 +37,8 @@
     "@glennsl/bs-jest": "^0.4.9",
     "bs-platform": "^5.0.6",
     "css-loader": "^3.2.0",
-    "html-webpack-plugin": "^3.2.0",
+    "favicons-webpack-plugin": "^3.0.1",
+    "html-webpack-plugin": "^4.0.0-beta.11",
```

Comment:
Why is this one beta now?

Reply:
Not sure, but the `favicons-webpack-plugin` wants this version of `html-webpack-plugin`.